AUTHORS

```diff
@@ -175,6 +175,9 @@ Julien Ammous <j.ammous at gmail.com>
 Kevin Bowling <kbowling at llnw.com>
  - write_tsdb plugin for http://opentsdb.net/
 
+Shirly Radco <sradco at redhat.com>
+ - write_syslog plugin.
+
```

Comment:
Sorry I missed this during earlier review, but keep this list sorted too please.

Reply:
Fixed